blogs/big_data/data_visualization/blog.md

```diff
@@ -0,0 +1,252 @@
+  
+# How to create a Tableau dashboard
+
+**This blog will teach you how to create a Tableau dashboard in the following steps:**
+* Step 1: Connect to a data source
+* Step 2: Create a table with filters
+* Step 3: Create a line graph with a date filter
+* Step 4: Format the dashboard
+
+Step 1: Connect to data source
+- 
+Please follow the link below for download. A 1-year free trial is available for students and teachers!
+
+https://www.tableau.com/products/desktop/download
+
+If you would like to download the dataset I used, please follow the link below and select New_York_Times >> covid-19-county-level-data.csv and convert the csv file into an excel file.
+
+https://www.kaggle.com/roche-data-science-coalition/uncover
+
+Once you open Tableau, you need to connect to a data source. 
+We will use a Microsoft Excel file for this dashboard.
+
+![](https://i.imgur.com/UUjUMPf.jpg)
+
+After connection, this is my raw data. I'm using data on COVID-19 cases by county here:
```

Comment:
Might be more interesting for the reader to explain why you chose COVID-19 cases, and maybe what other sorts of data sets they could use (if you have any ideas).